src/WebJobs.Script.WebHost/Controllers/ExtensionsController.cs

```diff
@@ -98,7 +105,7 @@ public async Task<IActionResult> GetJobs()
 
         public async Task<IActionResult> InstallExtension(ExtensionPackageReference package, bool verifyConflict = true)
         {
-            if (package == null)
+            if (package == null || _extensionBundleManager.IsExtensionBundleConfigured())
             {
                 return BadRequest();
```

Comment:
Same here. Split package == null and _extensionBundleManager.IsExtensionBundleConfigured().  Throw BadRequest with a message if extensionbundle is configured